baselib.go

```diff
@@ -458,12 +458,11 @@ func baseUnpack(L *LState) int {
 }
 
 func baseXPCall(L *LState) int {
-	fn := L.CheckFunction(1)
 	errfunc := L.CheckFunction(2)
-
+	L.Remove(2)
```

Comment:
@yuin I am nut sure if removing element is fine. But as fare as I understand at the end of `baseXPCall` all arguments are anyway removed from stack.

So removing is a bit simpler then manually duplicate all arguments. For performance there is no use because remove will be sure no improvement because remove will also copy values.